### DISCUSSION TITLE
Any plans for JSON-LD support?

### DISCUSSION THREAD

**Discussion by raulfdm** [jonasmerlin/astro-seo#60] [Ideas]
Hey... first, thanks for this project. It comes very handy while I'm migrating from Next to Astro.

I was wondering if you already have plans to support JSON-LD.

Of, do you already have a release/feature plan for this library?

**Comment by spokospace** [comment]
I'm also looking for that, I used useHead with JSON-LD before.
It will be good to add something like this also to Astro :)
